doc/guides/doc-style-guide.md

```diff
@@ -33,6 +33,11 @@ this guide.
 * Documents must start with a level-one heading.
 * Prefer affixing links (`[a link][]`) to inlining links
   (`[a link](http://example.com)`).
+* Don't use [non-breaking spcaes][] in headings.
+* Always use the appropriate closing punctuation for punctuation that has left/right variants.
+* Always use `-` as an mdash.
```

Comment:
This literally says "When you need an em dash, use an en dash instead."

Let's stick with the [Microsoft Style Guide on this](https://docs.microsoft.com/en-us/style-guide/punctuation/dashes-hyphens/emes) and the Chicago Style like we do for everything else.

* For an em dash, use the em dash character: — 
* ...or the em dash HTML entitiy `&mdash;` which renders thusly: &mdash; 
* Do not use the en dash or hyphen: - 

There's no need for any of this stuff about dashes to be in our style guide. We refer people to Microsoft Style Guide and Chicago Manual of Style so that we don't have to have a very long style guide.